### PR TITLE
LPS-169410 | master

### PR DESCRIPTION
**Pull request by Hanlf**
https://issues.liferay.com/browse/LPS-169410

@austinchiang and @david-gutierrez-mesa 
There are some modules involving the **asset library** and **site**, such as StructuredContent and Language.
I divided the headless tests according to whether the API is **assetLibraries** related: 
For example, `getAssetLibraryStructuredContentsPage` is under the StructuredContent module, but it is assetLibraries related, then I moved the test to Lima.
Another example, `getAssetLibraryLanguagesPage` is under the Language module, it is related to assetLibraries, I moved it to Lima as well.
Please let me know what you think, thanks

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
@Hanlf My opinion is that we should sort these files based on what the primary test object is, not what the secondary or tertiary test factors are. In the case of `getAssetLibraryStructuredContentsPage` you're primarily testing a (web content) structure which happens to be in an asset library so I think it can belong to echo. The same goes for `getAssetLibraryLanguagesPage`; since the language component is owned by Echo I think it's best they own it. Without looking at the code I'm fairly certain asset libraries/sites both use the same or similar language framework which is owned by Echo.

LMK what you think @david-gutierrez-mesa, I think we'll need to draw some lines here

**Comment by david-gutierrez-mesa** [comment]
Hi there! I agree with @austinchiang . We should place the test based on the primary objective. So feel free to assign them to Echo. @austinchiang just to add that you should check if you would need to add some of the components where we are moving these tests to Lima modules in order to be executed on your PR. But that's up to you.

**Comment by austinchiang** [comment]
Converting to draft until progress is made

**Comment by Hanlf** [comment]
Hi @david-gutierrez-mesa 
Based on the primary test object, I left these three tests to Echo, thanks!
`ExposeErcFromAssetLibraryForStructuredContentFolders.testcase`
`ExposeErcFromAssetLibraryForStructuredContents.testcase`
`ExposeLanguagesFromAssetLibraryAndSite.testcase`
cc/ @austinchiang

File: modules/apps/headless/headless-delivery/headless-delivery-test/src/testFunctional/tests/HeadlessDeliverySupportDifferentParameters.testcase [hunk 1]
```diff
@@ -3,7 +3,7 @@ definition {
 
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.main.component.name = "Headless";
+	property testray.main.component.name = "Lima Headless";
```
Comment (austinchiang) [inline modules/apps/headless/headless-delivery/headless-delivery-test/src/testFunctional/tests/HeadlessDeliverySupportDifferentParameters.testcase:6]: There are a few tests here that belong to Echo. Please find them and reclassify them

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please make changes and resend @Hanlf

**Comment by Hanlf** [comment]
@david-gutierrez-mesa Another test(HeadlessDeliverySupportDifferentParameters.testcase) is left to your team, thanks!
@austinchiang Regarding the names of the test cases, I'll try to keep them as they are, and I'll update them if needed when I have a better understanding of the tests.

**Review by austinchiang — APPROVED** [review]
LGTM

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**